### PR TITLE
[Makefile] Add target `make format`

### PR DESCRIPTION
**Pull request by ayushkovskiy**
very useful, we have it in almost all our python projects.

### CODE REVIEW

**Comment by AlekseySh** [comment]
I hate blake

**Comment by mariyadavydova** [comment]
@AlekseySh What's wrong with it?

**Comment by ayushkovskiy** [comment]
I think in Template we should follow best practices of programming, and black+lint are best practices. Since we have lint, I think we should have black as well.

File: {{cookiecutter.project_slug}}/Makefile [hunk 1]
```diff
@@ -196,10 +196,17 @@ kill: kill-training kill-jupyter kill-tensorboard kill-filebrowser
 setup-local:  ### Install pip requirements locally
 	$(PIP) -r requirements.txt
 
+.PHONY: format
+format:  ### Run static code analysis locally
```
Comment (mariyadavydova) [inline {{cookiecutter.project_slug}}/Makefile]: Incorrect help message. I assume this is "Automatically format the code" rather than run the analysis.

**Pull request merged**